mlflow/R/mlflow/R/tracking-runs.R

```diff
@@ -116,50 +116,24 @@ mlflow_get_run <- function(run_id = NULL, client = NULL) {
 #'   data may be written.
 #' @template roxlate-client
 #' @template roxlate-run-id
-#' @param metrics A named list of metrics to log.
-#' @param params A named list of params to log.
-#' @param tags A named list of tags to log.
-#' @param timestamps (Optional) A list of timestamps of the same length as `metrics`.
+#' @param metrics A dataframe of metrics to log, containing the following columns: "key", "value",
+#'  "step", "timestamp"
+#' @param params A dataframe of params to log, containing the following columns: "key", "value"
+#' @param tags A dataframe of tags to log, containing the following columns: "key", "value"
 #' @export
-mlflow_log_batch <- function(metrics = NULL, params = NULL, tags = NULL, timestamps = NULL,
-                             run_id = NULL, client = NULL) {
+mlflow_log_batch <- function(metrics = NULL, params = NULL, tags = NULL, run_id = NULL,
+                             client = NULL) {
```

Comment:
Should we verify that the dataframes contain mandatory columns (with no missing) and do not have any extra columns?

Reply:
Good point! Added validation!